service/src/main/java/uk/gov/hmcts/reform/fpl/model/CaseData.java

```diff
@@ -612,6 +605,7 @@ public DynamicList buildDynamicHearingList(UUID selected) {
     private final Object hearingDateList;
     private final Object pastAndTodayHearingDateList;
     private final Object futureAndTodayHearingDateList;
+    private final Object toReListHearingDateList;
```

Comment:
Thoughts on creating a `ManageHearingsEventData` class like I did with the `UploadCMOEventData`?

Reply:
Maybe (if you think that it is a worthwhile idea) better as a future tech debt, would be quite far out of the scope of this story